libs/classifiers/classifier.py

```diff
@@ -18,7 +18,8 @@ def inference(self, objects):
         return self.classifier.inference(objects)
 
     def object_post_process(self, object, classifier_result, classifier_score):
-        if object['face'] is not None and classifier_score > self.min_threshold:
-            object['face_label'] = classifier_result
-        else:
-            object['face_label'] = -1
+        if object.get('face') is not None:
```

Comment:
If you check for existence, please do it with `if 'face' in object` rather than using `.get()`, I THINK if someday having None as the value of the key 'face' will be meaningful, this breaks the code as it exists but returns None.